skimage/io/collection.py

```diff
@@ -434,10 +429,6 @@ class MultiImage(ImageCollection):
     will only read one frame, while `ImageCollection` by default will read
     all of them.
 
-    The current implementation of `MultiImage` is simply a small wrapper
```

Comment:
I don't see this text in the current file; is this PR based on the latest branch?

I see:

```
    The current implementation makes use of ``tifffile`` for Tiff files and
    PIL otherwise.
```

Which is also wrong (as a docstring), but probably describes the desired behavior.

Reply:
The text being removed here is from the first commit in this PR, not the `main` branch